test/agents/test_agent.py

```diff
@@ -64,7 +64,7 @@ def test_agent_chooses_no_action():
             description="useful for when you need to retrieve documents from your index",
         )
     )
-    with pytest.raises(AgentError, match=r"Wrong output format.*"):
+    with pytest.raises(AgentError, match=r"Could not identify the next tool from Agent's output.*"):
```

Comment:
If we change the error message, we will also need to update the string in the test to
`"Could not identify the next tool or tool input from Agent's output.*"`